.github/workflows/build-docs.yml

```diff
@@ -3,13 +3,20 @@ name: 'Build Documentation'
 on:
   push:
     branches: [ main, stable, oldstable ]
+  pull_request:
+    branches: [ main, stable, oldstable ]
 
 jobs:
   generate-doc-and-upload-coverage:
     name: Build XML documentation and upload coverage
     runs-on: ubuntu-latest
     container: greenbone/doxygen
     steps:
+      - name: Install gvmd doc dependencies
+        run: |
+          apt update
+          apt install --no-install-recommends -y xmltoman xsltproc
+          rm -rf /var/lib/apt/lists/*
```

Comment:
This isn't required. It's just for reducing the size of a image layer when publishing a container image.
```suggestion
```